ui/src/app/views/dataset-page/component.tsx

```diff
@@ -117,6 +136,9 @@ export const DataSetPage = withCurrentWorkspace()(class extends React.Component<
       selectedValues: [],
       openSaveModal: false,
       queries: [],
+      dataSets: [],
+      selectedPreviewDomain: '',
```

Comment:
I would probably try to default this to null, and type it as a `Domain`? I think it makes more sense as a typing than a free-form string.

Reply:
this is a string the data coming back as domain as a string value